test/test_normal_form_game.jl

```diff
@@ -141,6 +141,8 @@
         @test @inferred(payoff_vector(player, nothing)) == [0, 1]
         @test @inferred is_best_response(player, 2, nothing)
         @test @inferred(best_response(player, nothing)) == 2
+        @test is_dominated(player, 1) == true
```

Comment:
Maybe you can just do

```suggestion
        @test is_dominated(player, 1)
```